src/AmqpQueue.php

```diff
@@ -37,7 +37,7 @@ public function setArguments(array $arguments);
 
     /**
      * @param string $key
-     * @param string|bool|int|float $value
+     * @param mixed  $value
```

Comment:
Arguments is a `table` domain, and `table` can even contains other tables (quite rare cases, but anyway). Also, null is valid value for table field too.